src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,125 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static int initialCapacity = 16;
```

Comment:
The field `initialCapacity` should be a constant. It's a common mistake to forget to mark such fields as `final`. The name of the constant should be in uppercase. The same applies to the `initialLoadFactor` field.

Reply:
Done